r-package/grf/vignettes/grf.Rmd

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
Perhaps this duplicates the 'Usage Examples' under the landing page too much?

Reply:
I thought an exact duplicate was convenient because pkgdown would render it so you could directly see the output (without running it yourself)?